client/post-editor/editor-sidebar/feedback-request-form.jsx

```diff
@@ -0,0 +1,36 @@
+/**
+ * External dependencies
+ */
+import React, { PureComponent } from 'react';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import FormTextInput from 'components/forms/form-text-input';
+import Button from 'components/button';
+
+class FeedbackRequestForm extends PureComponent {
+	render() {
+		const { translate } = this.props;
+
+		const description = translate(
+			'Send your friends a link to read your draft before you publish.'
+		);
+
+		return (
+			<div className="editor-sidebar__feedback-request-form">
+				<p>{ description }</p>
+				<label>
+					{ translate( 'Friend\'s Email' ) }
```

Comment:
feel free to use double quotes or backticks here to avoid escaping; some have different opinions on what should happen here